qiskit/synthesis/evolution/qdrift.py

```diff
@@ -75,7 +74,7 @@ def synthesize(self, evolution):
         # The protocol calls for the removal of the individual coefficients,
         # and multiplication by a constant evolution time.
         evolution_time = lambd * time / num_gates
-        self.sampled_ops = algorithm_globals.random.choice(
+        self.sampled_ops = np.random.choice(
```

Comment:
This isn't ideal, because it removes a method of reproducibility.  We may want a new `seed` argument to the class, and for it to store its random state (`self.rng = np.random.default_rng(seed=seed)` or whatever).